requirements.txt

```diff
@@ -0,0 +1,15 @@
+beautifulsoup4==4.8.2
+bs4==0.0.1
+certifi==2019.11.28
+chardet==3.0.4
+docutils==0.16
+idna==2.9
+PyJWT==1.7.1
+python-dateutil==2.8.1
+pytz==2019.3
+requests==2.23.0
+six==1.14.0
+soupsieve==2.0
+statistics==1.0.3.5
+twilio==6.37.0
+urllib3==1.25.8
```

Comment:
How does python manage multiple requirements files in a multi-project repo? Should this go into a separate project folder?

Reply:
Good questions. Yeah, I think that makes sense. I'm going to take this opportunity to look into [Poetry](https://github.com/python-poetry/poetry).